.github/workflows/ExecuteProcedure_build_and_test_on_main.yml

```diff
@@ -0,0 +1,25 @@
+name: ExecuteProcedure_build_main
+
+on:
+  push:
+    branches:    
+      - main
+    paths:
+      - 'Frends.MySQL.ExecuteProcedure/**'
+  workflow_dispatch:
+
+jobs:
+  build:
+    uses: FrendsPlatform/FrendsTasks/.github/workflows/build_main.yml@main
```

Comment:
The Main workflow will also run tests, so you should use docker container as you have done in Push workflow.